src/extensions/updateHelper.ts

```diff
@@ -154,6 +155,25 @@ async function getConsent(field: string, message: string): Promise<void> {
   }
 }
 
+/**
+ * Displays an update warning as markdown, and prompts the user for confirmation.
+ * @param updateWarning The update warning to display and prompt for.
+ */
+export async function confirmUpdateWarning(updateWarning: UpdateWarning): Promise<void> {
+  logger.info(marked(updateWarning.description));
+  if (updateWarning.action) {
+    logger.info(marked(updateWarning.action));
+  }
+  const continueUpdate = await promptOnce({
+    type: "confirm",
+    message: "Do you wish to continue with this update?",
+    default: false,
+  });
```

Comment:
Question: do we need to bypass this prompt if we're not in interactive mode? Is this a case we need to consider? cc @laurenzlong

Reply:
We already do! Some smart person built that functionality into prompt.ts https://github.com/firebase/firebase-tools/blame/master/src/prompt.ts#L32

Reply:
But this codepath here does _not_ support that behavior. It's not even supported in `promptOnce` (though it is in `prompt`). It would have to be plumbed through a bit.

Reply:
You're correct, I totally misread this.

Reply:
We don't support non-interactive in ext:install currently. So it would be a more broad conversation, and out of scope of this PR. I'm also unsure about whether we would want to support non-interactive in extension installs/updates ever. When we went through security review, one of the things that we talked about was that the user would see the set of IAM permission and explicity agree to them. This would not be possible in non-interactive mode.

Reply:
That sounds reasonable to me